src/components/bmc/utils.ts

```diff
@@ -0,0 +1,57 @@
+import { AddBmcValues } from './types';
+
+export const getBareMetalHostCredentialsSecret = (
```

Comment:
shouldnt this live in `dynamic-cim` ?

Reply:
I think those methods are generic enough to be used by various consumers which will need that k8s resource composition in their flows.